README.md

```diff
@@ -63,7 +61,7 @@ module.exports.migrate = async ({firestore, FieldValue}) => {
 
 ## Running locally
 
-Typically, `fireway` expects a `--projectId` option that lets you specify the Firebase project associated with your Firestore instance against which it performs migrations. 
+Typically, `fireway` expects an `GOOGLE_APPLICATION_CREDENTIALS` environment variable to be set.
```

Comment:
Changing a `projectId` from `sample-prod` to `sample-staging` seems simpler than changing an env variable.